### DISCUSSION TITLE
Running an archive node on ETH

### DISCUSSION THREAD

**Discussion by letsbangout** [paradigmxyz/reth#11522] [General]
I started an ETH node with `reth node --http` on v1.0.6 afaik.  The beacon was started with `lighthouse bn --network mainnet --eth1 --port 10000 --datadir /root/beacondata --http --execution-endpoint http://127.0.0.1:8551 --execution-jwt /somepathto/jwt.hex --disable-deposit-contract-sync --checkpoint-sync-url https://mainnet-checkpoint-sync.stakely.io`.  Much to my surprise, when I queried the node with:

`curl --data '{"method":"eth_getBalance","params":["0xe5Fb31A5CaEE6a96de393bdBF89FBe65fe125Bb3", "0x1"],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST`

which is a standard query i'd run to determine if a given node has archival information, the node responded that it was unaware of this block.  It's my understanding that the L1 (reth) determines whether a given node is archival or not, and that reth defaults to archive/no pruning.  Any idea why I don't have an ETH mainnet archive?  Thx.

**Comment by Rjected** [comment]
Is the node synced yet?

**Reply by letsbangout** [reply]
No, currently sitting at `stage_progress=97.13%` according to the reth logs.
